pyupgrade/_plugins/capture_output.py

```diff
@@ -0,0 +1,97 @@
+import ast
+import functools
+from typing import Iterable
+from typing import List
+from typing import Sequence
+from typing import Tuple
+
+from tokenize_rt import Offset
+from tokenize_rt import Token
+
+from pyupgrade._ast_helpers import ast_to_offset
+from pyupgrade._ast_helpers import is_name_attr
+from pyupgrade._data import register
+from pyupgrade._data import State
+from pyupgrade._data import TokenFunc
+from pyupgrade._token_helpers import find_open_paren
+from pyupgrade._token_helpers import parse_call_args
+
+
+def _delete_stderr(
+    tokens: List[Token],
+    func_args: Sequence[Tuple[int, int]],
+    stderr_arg_idx: int,
+) -> None:
+    if stderr_arg_idx == 0:
+        del tokens[
+            func_args[stderr_arg_idx][0]: func_args[stderr_arg_idx + 1][0]
+        ]
+    else:
+        del tokens[
+            func_args[stderr_arg_idx - 1][1]: func_args[stderr_arg_idx][1]
+        ]
+
+
+def _replace_stdout(
+    tokens: List[Token],
+    func_args: Sequence[Tuple[int, int]],
+    stdout_arg_idx: int,
+) -> None:
+    tokens[slice(*func_args[stdout_arg_idx])] = [
+        Token('SRC', ' capture_output=True'),
+    ]
```

Comment:
I don't think there's any benefit to having these be free functions

that said, I think there might be some benefit to having a "remove argument" and "replace argument" helper for the function rewriting (actually it might already exist but you'd have to poke around?)

Reply:
My thinking for having them is that I need to first operate on the second one, and then on the first one - otherwise the token indices will get shifted. I'll see if I can find another way though